src/native/external/llvm-libunwind.cmake

```diff
@@ -9,13 +9,6 @@ set (LLVM_LIBUNWIND_SOURCES_BASE
     src/libunwind.cpp
 )
 
-if(CLR_CMAKE_TARGET_APPLE)
-    set(LLVM_LIBUNWIND_SOURCES_BASE
-        ${LLVM_LIBUNWIND_SOURCES_BASE}
-        src/Unwind_AppleExtras.cpp
```

Comment:
It was deleted in https://github.com/llvm/llvm-project/commit/1ae57fed8806886238b7603f6b3b54350da83f0e.